docs/source/tutorial/3-troubleshooting.ipynb

```diff
@@ -152,17 +152,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from pydra.utils.general import default_run_cache_root\n",
-    "import cloudpickle as cp\n",
-    "from pprint import pprint\n",
-    "from pydra.tasks.testing import Divide\n",
-    "\n",
-    "with open(\n",
-    "    default_run_cache_root / Divide(x=15, y=0)._checksum / \"_error.pklz\", \"rb\"\n",
-    ") as f:\n",
-    "    error = cp.load(f)\n",
+    "%%bash\n",
     "\n",
-    "pprint(error)"
+    "pydracli crash _error.pklz --rerun --debugger pdb"
```

Comment:
Could you leave the Python cell above that generates the crash file and then use that for the bash cell?